File: 프로그래머스/기초 트레이닝 - Level 0/날짜 비교하기-solve.java
class Solution {
    public int solution(int[] date1, int[] date2) {
        int[] days = {365, 30, 1};
        int total1 = 0;
        int total2 = 0;
        for (int i = 0; i < 3; i++) {
            total1 += days[i] * date1[i];
            total2 += days[i] * date2[i];
        }
        
        return total1 < total2 ? 1 : 0;
    }
}

File: 프로그래머스/기초 트레이닝 - Level 0/날짜 비교하기-solve.py
def solution(date1, date2):
    total1 = 0 
    total2 = 0
    for day1, day2, days in zip(date1, date2, [365, 30, 1]):
        total1 += days * day1
        total2 += days * day2
        
    return int(total1 < total2)